data/sql/updates/pending_db_world/rev_1720992304054237637.sql

```diff
@@ -0,0 +1,3 @@
+--
+UPDATE `creature_text` SET `Text` = '%s takes a deep breath!' WHERE `creatureId` = 21217;
+UPDATE `broadcast_text` SET `MaleText` = '%s takes a deep breath!' WHERE `ID` = '20021';
```

Comment:
```suggestion
UPDATE `creature_text` SET `Text` = '%s takes a deep breath!', `BroadcastTextId` = 0 WHERE `creatureId` = 21217;
```
Broadcast texts are not to be meddled with currently, as they've been bruteforced from Blizzard. It just so happens that they were taken from a Mists of Pandaria build where a bunch of texts (seems to include this one) were changed.